### DISCUSSION TITLE
follow_redirect middleware doesn't work with faraday 2.0

### DISCUSSION THREAD

**Discussion by jcoyne** [lostisland/awesome-faraday#6] [General]
It is a part of the faraday_middleware gem, which doesn't support faraday 2.0. Should it be removed from the readme on awesome-faraday?

**Comment by olleolleolle** [comment]
It should get an Issue reported and Pull Requests are welcome.

**Reply by jcoyne** [reply]
I don't understand what "It" refers to. Is it the faraday_middleware gem or awesome-faraday readme?  What direction should this go?

**Reply by olleolleolle** [reply]
Apologies for confusion, the middleware software is what I tried to refer to.  Now, it could be that it needs pulling out into its own gem, so as to support 2.0, which would make it clearer to people upgrading. 

Faraday_middleware should remain supporting the Faraday version it is. It represents the "everything baked in" way, which we are digging our way out of.
